cabal.project

```diff
@@ -6,6 +6,10 @@ packages:
   optics-th/*.cabal
   generic-optics/*.cabal
 
+-- Code generation for optic kinds
+packages:
+  codegen/*.cabal
```

Comment:
I specifically have `cabal.codegen.project`, so one don't need to support wide GHC window for utilities